pkgs/by-name/se/serie/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "serie";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "lusingander";
+    repo = pname;
```

```suggestion
    repo = "serie";
```
If I recall correctly `repo = pname;` is a bad practice.

---

You're absolutely right.